frontend/views/containers/proposals/Mincome.vue

```diff
@@ -50,16 +56,17 @@ export default {
         steps: [
           'GroupMincome'
         ]
-      },
-      fakeStore: {
-        groupCurrency: '$ USD'
       }
     }
   },
+  updated () {
+    console.log('j')
```

Comment:
Why is this here?

Reply:
Was just to help to debug something before... removed.